frontend/src/components/App.jsx

```diff
@@ -8,7 +8,7 @@ import RobotOverview from './pages/RobotOverview/RobotOverview';
 import RobotInteractionCockpit from './pages/RobotInteractionCockpit/RobotInteractionCockpit';
 
 /**
- * @description route component of the application
+ * @description Route component of the application
```

Comment:
So that this description starts with a capital R
![grafik](https://user-images.githubusercontent.com/38314662/119628070-0f91d580-be0d-11eb-9a8e-cc0d088ce3a6.png)